{{cookiecutter.project_slug}}/_/deployment/application/overlays/production/route.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Route
+metadata:
+  annotations:
+    kubernetes.io/tls-acme: "true"
+  name: {{ cookiecutter.project_slug }}
+spec:
+  host: example.com
```

Comment:
Shouldn't that be a Cookiecutter variable? Otherwise the demos won't habe a valid name.

Reply:
This is somewhat an edge case. I though of it as an actual _example_. I really didn't want to add another parameter to the setup and have to handle the "no domain" case. But, yeah, we shouldn't make compromises. I'll fix that.